eslint/babel-eslint-parser/src/worker/maybeParse.cts

```diff
@@ -13,13 +13,13 @@ let extractParserOptionsConfigItem: ConfigItem<any>;
 
 const MULTIPLE_OVERRIDES = /More than one plugin attempted to override parsing/;
 
-export = function maybeParse(
+export = async function asyncMaybeParse(
   code: string,
   options: InputOptions,
-): {
+): Promise<{
   ast: AST.Program | null;
   parserOptions: ParseResult | null;
-} {
+}> {
   if (!extractParserOptionsConfigItem) {
     extractParserOptionsConfigItem = babel.createConfigItemSync(
```

Comment:
`createConfigItemAsync` should be available after we modify `babel-core.cts`.

Reply:
added!